src/arviz_plots/plots/violin_plot.py

```diff
@@ -0,0 +1,331 @@
+    aes_by_visuals.setdefault("left_density", plot_collection.aes_set.difference("x"))
+    aes_by_visuals.setdefault("right_density", plot_collection.aes_set.difference("x"))
+    if "model" in distribution:
+        aes_by_visuals.setdefault("credible_interval", ["color", "x"])
+        aes_by_visuals.setdefault("point_estimate", ["color", "x"])
+    if "point_estimate" in aes_by_visuals and "point_estimate_text" not in aes_by_visuals:
+        aes_by_visuals["point_estimate_text"] = aes_by_visuals["point_estimate"]
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    # density
+    left_density_kwargs = copy(visuals.get("left_density", {}))
+
+    if left_density_kwargs is not False:
+        left_density_dims, left_density_aes, left_density_ignore = filter_aes(
+            plot_collection, aes_by_visuals, "left_density", sample_dims
+        )
+        default_color = plot_bknd.get_default_aes("color", 1, {})[0]
+        if "color" not in left_density_aes:
+            left_density_kwargs.setdefault("color", default_color)
+        with warnings.catch_warnings():
+            if "model" in distribution:
+                warnings.filterwarnings("ignore", message="Your data appears to have a single")
+            left_density = distribution.azstats.kde(
+                dim=left_density_dims, **stats.get("left_density", {})
+            )
+
+        # flip x and y coordinates
+        density_flipped = left_density.rename({"plot_axis": "flipped_plot_axis"})
+        density_flipped = density_flipped.assign_coords(flipped_plot_axis=["y", "x"])
+        density_flipped = density_flipped.rename({"flipped_plot_axis": "plot_axis"})
+
+        plot_collection.map(
+            line_xy,
+            "left_density",
+            data=density_flipped,
+            ignore_aes=left_density_ignore,
+            **left_density_kwargs,
+        )
+
+    right_density_kwargs = copy(visuals.get("right_density", {}))
+
+    if right_density_kwargs is not False:
+        right_density_dims, right_density_aes, right_density_ignore = filter_aes(
+            plot_collection, aes_by_visuals, "right_density", sample_dims
+        )
+        default_color = plot_bknd.get_default_aes("color", 1, {})[0]
+        if "color" not in right_density_aes:
+            right_density_kwargs.setdefault("color", default_color)
+        with warnings.catch_warnings():
+            if "model" in distribution:
+                warnings.filterwarnings("ignore", message="Your data appears to have a single")
+            right_density = distribution.azstats.kde(
+                dim=right_density_dims, **stats.get("right_density", {})
+            )
+
+        # flip x and y coordinates
+        density_flipped = right_density.rename({"plot_axis": "flipped_plot_axis"})
+        density_flipped = density_flipped.assign_coords(flipped_plot_axis=["y", "x"])
+        density_flipped = density_flipped.rename({"flipped_plot_axis": "plot_axis"})
```

Comment:
```suggestion
        density_flipped = left_density.sel(plot_axis=["x", "y"]).assign_coords(plot_axis=["y", "x"])
        density_flipped.loc[{"plot_axis": "x"}] = -density_flipped.sel(plot_axis="x")
```

You can get the data as negative like so instead of adding an argument to the visual function